client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,13 +347,23 @@ export class MySitesSidebar extends Component {
 	};
 
 	store() {
+		const allowedCountryCodes = [ 'US', 'CA' ];
```

Comment:
From a quick test, it seems this would not create problems - at least in sizes of the chunks. The woocommerce one went 0.01MB down, while the build one went 0.01MB up. This is for development build, on production the difference would have been even smaller. But the `getCountries` method doesn't seem the best match here - it returns a "lot" of data, not just the country codes. It'd be great to have one place with these country code, but I understand that this is still beta and in flux so it's good enough for now.